hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1074,6 +1074,75 @@ def hpat_pandas_series_max_impl(self, axis=None, skipna=True, level=None, numeri
     return hpat_pandas_series_max_impl
 
 
+@overload_method(SeriesType, 'prod')
+def hpat_pandas_series_prod(self, axis=None, skipna=None, level=None, numeric_only=None, min_count=0):
```

Comment:
```suggestion
def hpat_pandas_series_prod(self, axis=None, skipna=True, level=None, numeric_only=None, min_count=0):
```
According to documentation:
skipna : bool, default True

Reply:
None in Python is usually used as default value because Python consider default values as global values ([see this](https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument)).
Example:
```
def f(a=[]):
  a.append(1)
  return a

>>> f()
[1]
>>> f()
[1,1]
```
So common approach is:
```
def f(a=None):
   """Docstring: a: list, default []"""
  if a is None:
    a = []
  ...
```

Reply:
In this case with `bool` True is global and not mutable. But implementation of Pandas takes None as argument. Moreover Pandas is ready to get None so we also should be ready to get None and set it to True in the function.

So I think `skipna : bool, default True` is about logical value. It meas if you drop out `skipna` it will be considered as True.